src/lib/staged_ledger/staged_ledger.ml

```diff
@@ -282,13 +285,15 @@ module T = struct
         ~registers_begin:(Some snarked_registers)
         ~registers_end:
           { local_state = Mina_state.Local_state.empty ()
-          ; ledger =
+          ; fee_payment_ledger =
               Frozen_ledger_hash.of_ledger_hash (Ledger.merkle_root ledger)
+          ; parties_ledger = failwith "TODO"
```

Comment:
I think `parties_ledger` would have the merkle root of `ledger`, since it is the state after applying all the transactions in the scan state?